tests.lisp

```diff
@@ -23,7 +23,7 @@
   (run! 'spinneret))
 
 (defun visually-equal (string1 string2)
-  (let* ((lines1 (serapeum:lines string1))
+  (let ((lines1 (serapeum:lines string1))
          (lines2 (serapeum:lines string2)))
```

Comment:
This line need to be outdented to match the one above.

Reply:
I amended the commit. WDYT?